tests/tasks/test_celery.py

```diff
@@ -0,0 +1,10 @@
+def test_create_task(celery_app, celery_worker):
+    @celery_app.task
+    def multiply(x, y):
+        return x * y
+
+    # Force `celery_app` to register our new task
+    # See: https://github.com/celery/celery/issues/3642#issuecomment-369057682
+    celery_worker.reload()
+    assert multiply.run(4, 4) == 16
+    assert multiply.delay(4, 4).get(timeout=10) == 16
```

Comment:
Note to self, this uses Celery's pytest fixtures https://docs.celeryq.dev/en/stable/userguide/testing.html#function-scope

Reply:
There's a testing issue I'm having in the next PR where `celery_app` the Pytest fixture — which will automatically use the virtualised `celery_worker` instead of looking for the one via the broker (Redis, which isn't created as part of the test suite) — isn't being automatically used to process tasks invoked in tests.

That doesn't show up here since the new task is added to `celery_app` directly. I've circumvented it in that in the next PR by addressing the task directly from `celery_app.tasks[...]`, then calling that directly.